dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -63,9 +63,6 @@ exposeWebInfOnDispatch.desc=If true, a servlet can access files in the WEB-INF d
 decodeUrlPlusSign.name=Decode url plus sign
 decodeUrlPlusSign.desc=Decode the plus sign when it is part of the URL. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.decodeurlplussign. For servlet-5.0 and newer Servlet features, the default value is false.
 
-suppressHtmlRecursiveErrorOutput.name=Suppress recursive exception in the HTML response
-suppressHtmlRecursiveErrorOutput.desc=Suppresses the exception information from appearing in the HTML output when there is a recursive error that cannot be handled by an application's configured error page. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.suppressHtmlRecursiveErrorOutput.
-
```

Comment:
We should be removing it from all of the translations as well.

Reply:
from Phil,

Simply remove it from the English, the build should notice the changed English file and package it up for me, I will then send it to translations in the next shipment (due late December) and the translation team will remove the message from the translated files.
 

Philip Nickoll

(i.e just merge PR without any further action.)